codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -752,7 +752,7 @@ describePicture (Arc _ b e r) = printf "arc { startAngle = %4f , endAngle = %4f
 describePicture (ThickArc _ b e r w) = printf "thickArc { startAngle = %4f , endAngle = %4f , radius = %4f , width = %4f}" b e r w
 describePicture (Text _ txt) = printf "text { text = %s }" txt
 describePicture (Blank _) = printf "blank "
---describePicture (StyledText _ style font txt) = style font txt
+--describePicture (StyledText _ style font txt) = show " { style = %s , font = %s , txt = %s }" style font txt
```

Comment:
The font and style are not strings, so %s won't work with them directly.  However, you have a `Show` instance, so this would work if you just change `font` to `(show font)`, and `style` to `(show style)`